src/TritonCTS/test/src/find_clock_pad/Nangate45/Nangate45.lef

```diff
@@ -0,0 +1,12156 @@
+# 
```

Comment:
You should NOT be adding another copy of the contents of the Nangate45 directory to the  repo. 
As I pointed out in the monday zoom call, use a symbolic link. From doc/OpenRoadArch.md

The Nangate45 open source library data used by many tests is in
`test/Nangate45`. Use the following command to add a link in the tool
command

```
cd tool/test
ln -s ../../../test/Nangate45
```